src/Akeneo/Category/back/Infrastructure/Converter/InternalAPI/InternalAPIToStd.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Category\Infrastructure\Converter\InternalAPI;
+
+use Akeneo\Category\Application\Converter\ConverterInterface;
+use Akeneo\Category\Application\Converter\FieldsRequirementChecker;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class InternalAPIToStd implements ConverterInterface
+{
+    public function __construct(private FieldsRequirementChecker $fieldsRequirementChecker)
+    {
+    }
+
+    public function convert(array $data): array
+    {
+        $this->fieldsRequirementChecker->checkFieldsPresence(['code', 'labels'], $data['properties']);
```

Comment:
It could be a thing like this to be more readable : 
```suggestion
        $this->fieldsChecker->checkExist(['code', 'labels'], $data['properties']);
```

Reply:
I like it better my way :) Also your suggestion contradicts another one of your suggestions above (checkFieldsExist)